src/main/java/at/hannibal2/skyhanni/events/garden/pests/PestKillEvent.kt

```diff
@@ -0,0 +1,5 @@
+package at.hannibal2.skyhanni.events.garden.pests
+
+import at.hannibal2.skyhanni.api.event.SkyHanniEvent
+
+class PestKillEvent : SkyHanniEvent()
```

Comment:
```suggestion
object PestKillEvent : SkyHanniEvent()
```

as there is no arguments, this should be an object instead of a class